examples/src/main/java/alluxio/cli/RunTests.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli;
+
+import alluxio.AlluxioURI;
+import alluxio.client.ReadType;
+import alluxio.client.WriteType;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.options.DeleteOptions;
+import alluxio.examples.BasicNonByteBufferOperations;
+import alluxio.examples.BasicOperations;
+
+import java.util.Arrays;
+import java.util.List;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Driver to run Alluxio tests.
+ */
+@ThreadSafe
+public final class RunTests {
+  /**
+   * The operation types to test.
+   */
+  enum OperationType {
+    /**
+     * Basic operations.
+     */
+    Basic,
+    /**
+     * Basic operations but not using ByteBuffer.
+     */
+    BasicNonByteBuffer,
+  }
+
+  /** Read types to test. */
+  private static final List<ReadType> READ_TYPES =
+      Arrays.asList(ReadType.CACHE_PROMOTE, ReadType.CACHE, ReadType.NO_CACHE);
+
+  /** Write types to test. */
+  private static final List<WriteType> WRITE_TYPES = Arrays
+      .asList(WriteType.MUST_CACHE, WriteType.CACHE_THROUGH, WriteType.THROUGH,
+          WriteType.ASYNC_THROUGH);
+
+  private static void usage() {
+    System.out.println("Usage:");
+    System.out.println("(1) To run a predefined set of read/write tests:");
+    System.out.println(String
+        .format("java -cp %s %s <master address>", Version.ALLUXIO_JAR, RunTests.class.getName()));
```

Comment:
`getCanonicalName()` should be used, I think, for this particular usage, they are the same.
http://stackoverflow.com/questions/15202997/what-is-the-difference-between-canonical-name-simple-name-and-class-name-in-jav